pkg/capabilities/capabilities.go

```diff
@@ -112,35 +115,42 @@ func MergeCapabilities(base, adds, drops []string) ([]string, error) {
 		return nil, err
 	}
 
-	// Make sure that capDrop and capAdd are distinct sets.
+	if stringInSlice(All, capDrop) {
+		// "Drop" all capabilities; return what's in capAdd instead
+		return capAdd, nil
+	}
+
+	if stringInSlice(All, capAdd) {
+		// "Add" all capabilities;
+		return capabilityList, nil
+	}
+
+	for _, add := range capAdd {
+		if stringInSlice(add, capDrop) {
+			return nil, errors.Errorf("capability %q cannot be dropped and added", add)
+		}
+	}
+
```

Comment:
On the fence, should we do a similar test up higher to see if "ALL" was passed in to both drop and add?

Reply:
I think we can ignore that case.  Two weird to worry about.